resources/README.md

```diff
@@ -9,11 +9,15 @@
 
 ##<a name="section1"></a>Hive clients
 
+# NGSI Notifications
 A set of Hive clients mimicing the native Hive CLI written in several programming languages both for HiveServer1 and HiveServer2. Specifically, there are clients for:
 
 * Java:
     * [`hiveserver1-client`](./hiveclients/java/hiveserver1-client)
     * [`hiveserver2-client`](/.hiveclients/java/hiveserver2-client)
+* Python:
+    * [`hiveserver1-client.py`](./hiveclients/java/hiveserver1-client.py)
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/3da5ee39ebc1b9baf11128573c3db2e1d277669c